serverlessworkflow/modules/ROOT/pages/cloud/quarkus/build-workflow-images-with-tekton.adoc

```diff
@@ -1 +1,167 @@
-//= Building Workflow Images with Tekton Pipelines
\ No newline at end of file
+= Building Workflows Images with Tekton
+:compat-mode!:
+// Metadata:
+:description: Building Workflow Images with Tekton
+:keywords: kogito, workflow, serverless, kubernetes, minikube, openshift, containers, tekton, pipeline, ci
+
+This document describes how to build a Workflow Application on a Kubernetes cluster using the Tekton pipeline.
+
+
+.Prerequisites
+You have a git repository containing your workflow and the Containerfile to build the workflow as a container image
+and a namespace where you can install Tekton i.e. `custom-namespace`.
+
+.OptionalPrerequisites
+Tekton CLI:
+
+- On Openshift in the Operator pipelines is available a link to download the correct cli from the operaotr
+- On Minikube/kind/Kubernetes use this link https://tekton.dev/docs/cli/
```

Comment:
move the tekton link to a var, e.g. tekton_cli_url
and refer here as link:{tekton_cli_url}[link].